src/platform/nrfconnect/CHIPPlatformConfig.h

```diff
@@ -106,21 +106,16 @@
 #define CHIP_CONFIG_LOG_MODULE_Support_PROGRESS 0
 #endif
 
-// Set MRP retry intervals for Thread and Wi-Fi to test-proven values.
 #ifndef CHIP_CONFIG_MRP_LOCAL_ACTIVE_RETRY_INTERVAL
-#if CHIP_ENABLE_OPENTHREAD
-#define CHIP_CONFIG_MRP_LOCAL_ACTIVE_RETRY_INTERVAL (800_ms32)
-#else
+#ifndef CONFIG_NET_L2_OPENTHREAD
```

Comment:
Do we need it, or should leave default values for Wi-Fi as well?

Reply:
You mean 300ms? I thought we had a reason for increasing it on Wi-Fi so didn't want to change it.

Reply:
Oh, i thought that we set 2s for both Thread and Wifi by default, and here we set 1s for Wi-Fi.. But this is not the case.